src/main/resources/db/migration/V2024_09_24_1500__CIV-15381_contact_court_welsh_amends.sql

```diff
@@ -0,0 +1,6 @@
+
+/**
+ * Add task item template
+ */
+update dbs.task_item_template set task_name_cy = replace(task_name_cy, 'Contact the court to request a change to my case', 'Cysylltu â''r llys i ofyn i wneud newid i fy achos')
+where task_name_cy like '%Contact the court to request a change to my case%';
```

Comment:
Should we not be doing an exact match if we know the ID ? Just in case this is included in another name somewhere.

Reply:
@ss-evoco I have updated the where clause. Please have a look